### PR TITLE
Library managment

### PR DESCRIPTION
**Pull request by rohitg-webonise**

### CODE REVIEW

File: calculator-react-code/src/App.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          {history.map((item, index) => (
+            <HistoryItem key={index} item={item} />
+          ))}
+        </ul>
+      </div>
+    </div>
+  );
+};
+
+const App = () => (
```
Comment (apeksha-haptiq) [inline calculator-react-code/src/App.js:112]: Pls remove calculator code

File: library-managment-system/src/Details/SignInPage.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// src/pages/SignInPage.js
+import React from "react";
+import LoginForm from "../Components/LoginForm";
+import "../style.css";
+
+const SignInPage = ({ onSignIn }) => {
```
Comment (apeksha-haptiq) [inline library-managment-system/src/Details/SignInPage.js:6]: Have not implemented sign out functionality

File: library-managment-system/src/App.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
+// src/App.js
+import React, { useState } from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
```
Comment (apeksha-haptiq) [inline library-managment-system/src/App.js:3]: In v6.4, new routers were introduced that support the new data APIs, should have used createBrowserRouter according to the hint

File: calculator-react-code/src/App.js (hunk 1 above)
Comment (rohitg-webonise) [reply calculator-react-code/src/App.js:112]: Done In New pr

File: library-managment-system/src/Details/SignInPage.js (hunk 2 above)
Comment (rohitg-webonise) [reply library-managment-system/src/Details/SignInPage.js:6]: Not yet. Do we need to implement the signOut functionality as well ?

**Pull request closed without merging**